src/site/content/en/lighthouse-pwa/load-fast-enough-for-pwa/index.md

```diff
@@ -21,26 +20,26 @@ Checklist](https://developers.google.com/web/progressive-web-apps/checklist#base
 
 ## Recommendations
 
-There are two main metrics regarding how users perceive load time:
+There are two main metrics that affect how users perceive load time:
 
-- The page appears visually complete.
-- The page is interactive. If a page appears visually complete at 1s, but the user can't interact with it until 10s, then the perceived page load time is 10s.
+- Time until page appears visually complete.
+- Time until page is interactive. If a page appears visually complete after 1 second, but the user can’t interact with for 10 seconds, the perceived page load time is 10 seconds.
```

Comment:
should it be "can't interact with it for 10 seconds" ?

Reply:
Yes, thanks for the catch.